core/src/main/java/org/apache/iceberg/puffin/StandardBlobTypes.java

```diff
@@ -26,4 +26,17 @@ private StandardBlobTypes() {}
    * href="https://datasketches.apache.org/">Apache DataSketches</a> library
    */
   public static final String APACHE_DATASKETCHES_THETA_V1 = "apache-datasketches-theta-v1";
+
+  /**
+   * A serialized form of HIVE_COLUMN_STATS_OBJ. The full list of available stats are provided in the
```

Comment:
rather than saying `HIVE_COLUMN_STATS_OBJ`, should this say `Hive column stats object`?

Reply:
Done, I have updated the same to  "Hive column stats object".